charts/bitwardenrs/1.1.1/values.yaml

```diff
@@ -0,0 +1,178 @@
+# Default values for bitwardenrs.
```

Comment:
You basically need to copy this to test_values.yaml to run the CI.
Some things might need to be  enabled/disabled depending on weither you are running the CI or a TrueNAS SCALE system, because you'll discover down the road :)